commons/jest.config.json

```diff
@@ -0,0 +1,7 @@
+{
+  "preset": "ts-jest",
+  "silent": true,
+  "testEnvironment": "node",
+  "testMatch": ["**/*.spec.(ts)"],
+  "verbose": true
```

Comment:
https://github.com/well-known-components/template-server/blob/main/jest.config.js#L11-L12

Please add coverage reports to all `jest.config` calls, the `--coverage` arg is needed to make it work.

```
  coverageDirectory: "coverage",
  collectCoverageFrom: ["src/**/*.ts", "src/**/*.js"],
```

Reply:
Added 👍